starboard/android/apk/app/src/main/java/dev/cobalt/media/MediaCodecCapabilitiesLogger.java

```diff
@@ -0,0 +1,216 @@
+// Copyright 2017 The Cobalt Authors. All Rights Reserved.
```

Comment:
nit: it should be 2023 instead of 2017.

Reply:
The code is copied from MediaCodecUtil.java, where we have copyright since 2017.